python/cudf/cudf/_lib/column.pyi

```diff
@@ -74,6 +74,10 @@ class Column:
     def has_nulls(self) -> bool:
         ...
 
+    @property
+    def has_nans(self) -> bool:
```

Comment:
A bit concerned about making this a property of the `Column` base class, as it's specific only to floating dtypes.

Reply:
As discussed offline moved this logic to `NumericalColumn`

Reply:
Since this is now a property onlyof `NumericalColumn`, we shouldn't include the typing info for it here. Could just be in `NumericalColumn`?

Reply:
Cleaned up 👍